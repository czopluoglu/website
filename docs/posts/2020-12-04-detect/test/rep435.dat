00010010100011101111
11001000001010000110
00100101101010010010
00100000001100000110
10100100011000000011
10100011001111000111
00110000001111110111
01000001011101111010
00101001111000000011
01110010111010110111
10111001101010101110
01101100110010001101
10001110011101111111
00000001011111000110
00101011001010100011
00101111111100101011
11110111111000000010
10011011011000100100
10111101101001001000
00000001011111100101
11001001111111010010
11010101111011111111
00010100011100011100
00101001001111011010
00011100111110101001
11110111110101001011
11010101100000010111
01001001001110111111
10101001100111100111
00111100001111111100
11000001011110000001
00000001101101111111
10110001011111001011
11111110001110001110
11011011101010010010
10000100011000110010
10111111111100110110
10011101111001110111
10100110011110011011
11110110110100101110
11110011111011101100
10000011011111100011
11100011000101110011
00100000001010011110
00110101001111101111
10111001111010100011
01000000100010000000
00001000110100000000
10110011011101000011
00100000001110101011
01000101011010111111
00011110101011101111
11010111010011101101
11010111111000010111
01011000001001111110
01001010101011110101
01001011001011010010
01101001111010111100
01111111111110110100
01110001010011110010
01100011101011011110
11001111011110100110
00000000101100010111
01101101001110111111
00001111011100111111
00101000010010010111
00001101101010010111
11111111110110110111
10111101101011111111
11101100101111111111
10110111111111011111
01100110001101000111
11110001011111100111
11111010010011010110
00011100010001111011
00111110100000110010
10010000001110101011
10111111111111111111
11011000001000000110
01111111111011111111
11000000101110100111
10110100011000001001
00000010001010111000
11011001000110010000
01000111001111101111
00000001000011100111
10101001110011101111
11001101100001000000
01100110111010000100
11001101110000111000
11000001110110000110
11111111111111111111
01101010111010000110
11011000011100000100
00011000011111011111
00101100011010010100
01110110111110101111
00100011100010100111
10101001111110001110
00101000000000000101
01000100101010000000
00110001011111110111
01100110111101101111
11111111111111010111
01011110001010100010
00000100101000111010
01000101011111001110
00010010111010000011
10011001011010100001
10100100000110100101
11001001100000100111
10000000101010110011
10001111101010111010
00100111100011100011
01010101101000100001
11000011111110000111
10011111111111111111
11100111100000010001
01110000111100101011
01000110011011010111
11001001101101111110
10001101110011100001
11100100111101001011
10110100010010001010
11111100111010100101
10101100100010111110
01101000011000011110
11111100011011000111
00100100000011101000
11111111110110100101
10000001111010001011
01001011110001100100
11111011101011100110
11101110111010100110
00011100011101111111
01000111111110111111
00010100000111111010
00111011110100100011
11111001001010100111
11011011110000100101
01000101111000000010
01100111111011100011
01000001111001100101
00101000000110011010
10011011111111000111
11111100000010101111
10111111111010111011
00000000101110111010
11111111111111111001
01001000001100010111
10111000011111011011
01000011101010111111
01101110011010100110
10011001111111101011
11001000011100111011
01010111111100001110
10000101010010110011
11001001000110011010
10000101111000101010
11011111111100011111
00000001000010000110
11110100111000111110
10010001101111110010
11000111111110101111
00010001000010111111
00110101101011100110
11100100111000011111
10100001101101110011
00011000000010010010
10000111111111011111
10100101001111111111
10111011111110111110
00100000011011000010
01110111110100101110
11000111111110111111
11101010000001111110
10101101110100011100
01001011001010110101
00010110101010100011
11000111011100111011
10111011100100001110
00001000000011000110
00000100101111001101
01010101111000111000
11010110101111111111
10000101111010101010
10010001001100011111
11000000010000000010
00011101011011110001
10111011101110001000
10110001010001100110
10010110111001111111
00000101110011100100
01001100101100110110
10001110101110011011
00001001000100001011
11101101011010111110
01011101010100101110
11111110110101101110
11110001110000001000
11000010011000101111
01011011011000100010
00001000001100110000
11000000111100010111
11001001111111101010
00100111110001110010
11110111110101101100
00011011111001011010
10100111011111100001
11110100011010001111
11111110001100111111
10001101011110100111
01011000110010101011
10000110111100000110
11101101111101111101
00100000100111111011
11110110111001001010
10111101110110100110
01100110001001010111
01110000001110111010
01111011100010000111
10000001000001110001
11011001111011101111
10101011001011111011
00000101111011111111
10111100111100011011
11110111100000100010
11110111111010100011
10111111110111111111
00001100011100101010
10100001110100110111
01111110100010000011
10010100001001011100
00101001111110101111
10001110110110100110
11111111000110100111
10011000110000100110
01011011110110110110
10010111111010011111
00011000000010100010
11010111100011101111
00010001100011111001
10110011101000010000
10110000111001001011
11001110111000010110
10000100001110000111
00110000011010011011
11101111101101101000
00100111111111111111
00111001110111100111
