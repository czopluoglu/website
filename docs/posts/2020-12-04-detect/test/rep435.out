-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep435.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.40

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.05
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.9632
                   IDN index value:   0.7526
                           Ratio r:   0.6874

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    1    1    2    1    1    3    1    1
    4    4    3    3    3    4    4    2    4    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    3    4    6    7    9   10

   -----------CLUSTER  2-------------

    5   18

   -----------CLUSTER  3-------------

    8   13   14   15   20

   -----------CLUSTER  4-------------

   11   12   16   17   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++++++++-+-----+---
   +d-+++++++-----+----
   +-d++++----++------+
   +++d-++++---------+-
   +++-d+++-----+-+--+-
   +++++d+--++----+---+
   ++++++d+--+--+------
   ++-++-+d--+----+----
   ++-+----d+-+-+---+--
   -+---+--+d-++-+-++-+
   +----+++--d+++--+---
   --+-----+++d++++---+
   --+------+++d++-+++-
   ----+-+-+-+++d+--++-
   ---------+-+++d++--+
   -+--++-+---+--+d++++
   +--------++-+-++d+++
   --------++--++-++d++
   ---++-------++-+++d+
   --+--+---+-+--+++++d
 
 No cross validation for this DETECT run
